UpdateProfiles.plist

```diff
@@ -216,13 +216,13 @@
 		<key>SafariMojave</key>
```

Comment:
I think the name should be updated to MojaveSupplemental, despite the rather unorthodox package name.

Reply:
Renamed to SupplementalMojave just to be consistent with the other names.